### PR TITLE
Metadata and text changes

### PR DESCRIPTION
**Pull request by afoonalex**
1. I phrased things a bit differently - both in meta aread and in the
actual main page
2. Also - removed the last 2 "features" from the list of 8 (and
rearranged it to be 3 in a row).
3. Changed the Demo button to be "See plans" link

Not all was discussed and asked for so feel free to comment or decline
:)

### CODE REVIEW

File: website/_layouts/default.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <meta name="description" content="Make Your Company Data Driven. Connect to any data source, easily visualize and share your data.">
+    <meta name="description" content="Make Your Company Data Driven. Connect to any data source, easily query and visualize in moments and share your insights with your team via dashboards and integrations. Query data from multiple sources with our unique Query Results data source. Redash plays with many databases and services including Amazong Redshift, BigQuery, PotsgreSQL, MySQL, TreasureData, MS SQL Server, MongoDB, ElasticSearch, Hive, Google Spreadsheets, Impala, InfluxDB, Presto, ScyllaDB, Cassandra, Amazon DynamoDB, Python, Vertica, Graphite, Stitch. ">
```
Comment (arikfr) [inline website/_layouts/default.html]: I don't think we should list all the data sources here. It should be tags, no?

Comment (afoonalex) [inline website/_layouts/default.html]: tags should be useless but I can duplicate and add a tags/keywords section

**Comment by afoonalex** [comment]
Do you have any other comments? Any objections?

File: user-guide/queries/query-google-spreadsheets.md [hunk 2]
```diff
@@ -5,5 +5,5 @@ To load your spreadsheet in Redash share your it with your ServiceAccount’s em
 The query format is “DOC_UUID|SHEET_NUM” (for example “kjsdfhkjh4rsEFSDFEWR232jkddsfh|0”) - this will be the equivalent of "select * from db" type of query and will show you the entire table.
 
 To apply some manipulation on top of that data, you can do one two options:
-1. Use the "Query Results" data source and your query id as a table (select ..... from query_123), then you'll need to query it with SQLite syntax. Read more about query results as data sources [here](queries/using-datasets-as-data-sources.md).
+1. Use the "Query Results" data source and your query id as a table (select ..... from query_123), then you'll need to query it with SQLite syntax. Read more about query results as data sources [here](using-datasets-as-data-sources.md).
```
Comment (arikfr) [inline user-guide/queries/query-google-spreadsheets.md]: Quote the query with `, so it's nicely formatted.

File: website/_layouts/default.html (hunk 1 above)
Comment (arikfr) [inline website/_layouts/default.html]: It needs to be shorted anyway, how about something like: "Use Redash to connect to any data source (Redshift, BigQuery, MySQL, PostgreSQL, MongoDB and many more supported), query, visualize and share your data to make your company data driven."?

Comment (afoonalex) [inline website/_layouts/default.html]: I can wait, I guess :(

Comment (arikfr) [inline website/_layouts/default.html]: ?

File: website/index.html [hunk 3]
```diff
@@ -9,13 +9,12 @@
       <div class="container">
           <div class="row">
               <div class="col-md-12 text-center">
-                  <h2>Make Your Company Data Driven</h2>
-                  <h3>Connect to any data source, easily visualize and share your data</h3>
+                  <font color="F2F9FC"><h2>Query, Visualize and Share Data</h2>
+                  <h3>Connect to any data source, easily query and visualize in moments.<br/> Share your insights with your team via dashboards and integrations.<br/> Make your company data driven.</h3></font>
```
Comment (afoonalex) [inline website/index.html]: I can wait, I guess :(

File: website/index.html [hunk 4]
```diff
@@ -9,13 +9,12 @@
...
+                  <font color="F2F9FC"><h2>Make Your Company Data Driven</h2>
+                  <h3>Connect to any data source, easily visualize and share your data</h3></font>
                   <p id="buttons">
-                      <a href="https://demo.redash.io/" data-track data-track-location="hero" data-track-event="Clicked Demo"  class="btn btn-primary btn-lg btn-redash">Demo</a>
-                      <a href="/#get-started" data-track data-track-location="hero" data-track-event="Clicked Get Started" class="btn btn-primary btn-lg btn-redash">Get Started</a>
-                  </p>
-
+                      <a href="/#get-started" data-track data-track-location="hero" data-track-event="Clicked Get Started" class="btn btn-primary btn-lg btn-redash">Get Started</a></p>
+                      <a href="https://demo.redash.io/" data-track data-track-location="hero" data-track-event="Clicked See Plans"><font color="FFFFFF">See Plans</font></a>
+<p></p>
```
Comment (arikfr) [inline website/index.html]: Let's revert this as well - before dropping the demo link I would like us to think of metric to track to see if it had impact (positive or negative).

File: website/index.html [hunk 5]
```diff
@@ -9,13 +9,12 @@
       <div class="container">
           <div class="row">
               <div class="col-md-12 text-center">
-                  <h2>Make Your Company Data Driven</h2>
-                  <h3>Connect to any data source, easily visualize and share your data</h3>
+                  <font color="F2F9FC"><h2>Make Your Company Data Driven</h2>
```
Comment (arikfr) [inline website/index.html]: This seems to have no effect? Anyway shouldn't use font tags, but rather the stylsheet.

Comment (arikfr) [inline website/index.html]: Before you mentioned I didn't see any difference while comparing between two tabs :-)

**Pull request merged**